.ci/opensearch/Dockerfile

```diff
@@ -1,10 +1,17 @@
-ARG OPENSEARCH_VERSION
+ARG OPENSEARCH_VERSION=latest
```

Comment:
Could you please check how can we make this dynamic? We can't have static typing here.

Reply:
Could you please check if we need provide like [this](https://github.com/opensearch-project/opensearch-py-ml/blob/main/.ci/run-opensearch.sh#L78-L79) to [here](https://github.com/opensearch-project/opensearch-py-ml/blob/main/.ci/run-repository.sh#L27-L31)?

Reply:
> Could you please check how can we make this dynamic? We can't have static typing here.

So, the `latest` will be the default value if we don't pass `--build-arg`. 

 

> Could you please check if we need provide like [this](https://github.com/opensearch-project/opensearch-py-ml/blob/main/.ci/run-opensearch.sh#L78-L79) to [here](https://github.com/opensearch-project/opensearch-py-ml/blob/main/.ci/run-repository.sh#L27-L31)?

We don't need to change anything for the `Dockerfile.client` as we are not using python version to do anything.